app/models/asset.rb

```diff
@@ -1,6 +1,8 @@
 require "virus_scanner"
 require "services"
 
+# Mongoid doesn't seem to have find_each so rubocop would break it
+# rubocop:disable Rails/FindEach
```

Comment:
If this will never work in this app because Mongoid is weird, could we globally disable it (in `.rubocop.yml`) with the same comment? There may be some disadvantages, like if we ever switch this app away from Mongoid to a different ORM that doesn support it, we might forget to re-enable this cop?